src/lang/mongolog/mongolog.pl

```diff
@@ -135,7 +135,13 @@
 	% TODO: make the client return docs in a format that it accepts as input.
 	maplist(format_doc, Docs, Docs0),
 	maplist(bulk_operation, Docs0, BulkOperations),
-	mng_get_db(DB, Coll, Key),
+
+	%% check for prefix in the Key
+	once(((setting(mng_client:collection_prefix, Prefix),
```

Comment:
Maybe it would be a bit clearer and you wouldn't have to use once if you use implication here

Reply:
@AbhijitVyas ?